supervisors/sensitivity_analysis_morris/morris.R

```diff
@@ -0,0 +1,107 @@
+require(methods)
+library("rjson")
+library("sensitivity")
+library("httr")
+
+source("supervisors/sensitivity_analysis_morris/scalarmapi.R")
```

Comment:
Niestety nie znalazłam w miarę szybkiego sposobu żeby to naprawić. Myślę że dałoby się to zrobić znajdując ścieżkę do aktualnego pliku a później wyszukać w tym samym katalogu jednak nie wydaje mi się żeby to było dość proste. Czy można zostawić tak jak jest?